server/routes/eligibility-checker/cannot-continue.route.js

```diff
@@ -40,21 +37,11 @@ const handlers = {
   }
 }
 
-const _getContext = async request => {
-  const containsElephantIvoryIdk =
-    (await RedisService.get(request, RedisKeys.CONTAIN_ELEPHANT_IVORY)) ===
-    Options.I_DONT_KNOW
-
+const _getContext = async () => {
   return {
     pageTitle: 'You cannot continue',
-    helpText1a: `To use this service, you must know for sure whether your item ${
-      containsElephantIvoryIdk
-        ? 'contains elephant ivory.'
-        : 'qualifies for exemption.'
-    }`,
-    helpText1b: containsElephantIvoryIdk
-      ? 'If you’re uncertain about your item and you choose to declare it anyway, we’ll assume it does contain elephant ivory.'
-      : '',
+    helpText1a: 'To use this service, you must know for sure whether your item qualifies for exemption.',
+    helpText1b: 'If you’re uncertain about your item and you choose to declare it anyway, we’ll assume it does contain elephant ivory.',
```

Comment:
If I'm reading this correctly, this route returns entirely static content now so it should just sit in the view file

Reply:
Yeah. That make sense. I'll make the change

Reply:
@ben-sagar I've updated this now. Everything, apart from the pageTitle (which is used for Analytics), moved to the view